package.json

```diff
@@ -140,6 +140,7 @@
     "babel-plugin-dynamic-import-node": "^2.3.3",
     "babel-plugin-emotion": "^10.0.20",
     "babel-plugin-macros": "^2.7.0",
+    "babel-plugin-polyfill-corejs3": "^0.0.7",
```

Comment:
This needs to be moved to https://github.com/storybookjs/storybook/blob/next/lib/core/package.json

Reply:
good point, will move it